<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmEditName</class>
 <widget class="QWidget" name="frmEditName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/title.png</normaloff>img/title.png</iconset>
  </property>
  <widget class="QLineEdit" name="txtContent">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>23</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>70</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取  消</string>
   </property>
  </widget>
  <widget class="QLabel" name="labTitle">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>26</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSure">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>70</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确  定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
